queries.sql

```diff
@@ -130,4 +130,89 @@ WHERE filtered.animals_owned =
       JOIN animals ON owners.id = animals.owner_id
       GROUP BY owners.name
     ) AS filtered
-);
\ No newline at end of file
+);
+
+-- Project 4
+
+SELECT vets.name, animals.name, visits.date_of_visit  
+FROM animals
+JOIN visits ON visits.animal_id = animals.id
+JOIN vets ON visits.vets_id = vets.id
+WHERE vets.name = 'William Tatcher'
+ORDER BY visits.date_of_visit DESC
+LIMIT 1;  
+
+CREATE VIEW Stephanie_Mendez AS
+  SELECT animals.name, COUNT(animals.name) 
+  FROM animals
+  JOIN visits ON visits.animal_id = animals.id
+  JOIN vets Ve ON visits.vets_id = vets.id
```

Comment:
- [ ] You have a typo `VE`, kindly correct it before you merge. 👍

Reply:
Fixed. Thank you very much. This shows how thorough you are in your review.